### PR TITLE
LIS page typo fix/clarity enhancements for proofs

### PR DESCRIPTION
**Pull request by topshenyi-web**

### CODE REVIEW

**Comment by topshenyi-web** [comment]
Wait actually to make things clearer why not just make the definition of L_i[j] to be the smallest element from the first i elements with an increasing sequence of length j ending on it? (instead of j+1)

**Review by SansPapyrus683 — APPROVED** [review]

File: content/4_Gold/LIS.mdx [hunk 1]
```diff
@@ -113,26 +113,26 @@ Thus, $L_i[j]$ has to be greater than $L_i[j - 1]$. From that, we can write this
 equation: $L_i[j - 1] \leq a_{j-1} < a_j = L_i[j]$, which is exactly what we
 wanted.
 
-**Lemma 2:** The length of the LIS ending on $A[i + 1]$ is equal to the least
-index $j$ (with 0-indexing) such that $L_i[j] \geq A[i + 1]$.
+**Lemma 2:** The length of the LIS ending on $A[i + 1]$ is equal to $j+1$, where $j$ is the least
```
Comment (dongliu0426) [inline content/4_Gold/LIS.mdx:116]: length of LIS is index?

Comment (topshenyi-web) [reply content/4_Gold/LIS.mdx:116]: I thought the length was index+1?
"Let $L_i$ be an array (0-indexed) where $L_i[j]$ is the smallest element from
the first $i$ elements of $A$ with **an increasing sequence of length $j + 1$
ending on it** (or $\infty$ if there is no such element)."

Comment (topshenyi-web) [reply content/4_Gold/LIS.mdx:116]: then again, that's kind of confusing to begin with... maybe I should just change the definition instead

**Pull request closed without merging**